src/libraries/OSGroups.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
that file doesn't belong into the root of libraries. It's not libraries specific at all. Would eng\ or eng\configurations be a better place for it?

Reply:
We are currently  just using it in libraries and we are not even sure if it will remain in the future. There was as discussing regarding using rids which could be understood directly by sdk and nuget. If we implement that, we wont require this file

Reply:
I still think this belongs into the eng dir which consists of infrastructure files. src/libraries should ideally only consist of projects.